jcommune-model/src/main/java/org/jtalks/jcommune/model/entity/Topic.java

```diff
@@ -200,6 +200,7 @@ public void addPost(Post post) {
      * @param postToRemove post to remove
      */
     public void removePost(Post postToRemove) {
+        recalculateModificationDate();
```

Comment:
`recalculateModificationDate()` is called in `TransactionalPostService#deletePost` we should not duplicate this call. You should investigate why this method was not called in `TransactionalPostService#deletePost` and fix.